core-web/libs/template-builder/src/lib/components/template-builder/components/template-builder-theme-selector/template-builder-theme-selector.component.html

```diff
@@ -0,0 +1,77 @@
+<div class="theme__header">
+    <p-dropdown
+        #siteSelector
+        [filter]="true"
+        [showClear]="true"
+        [showClear]="false"
+        (onChange)="siteChange($event.value)"
+        dotSiteSelector
+        optionLabel="name"
+        filterBy="name"
+    >
+    </p-dropdown>
+
+    <div class="theme-search-box">
```

Comment:
BEM `header__theme-search-box`

Reply:
The box, according to line 1 is `theme`. Also in line 1: `theme__header` I think is bad for a block is should be `theme-header`